src/test/java/io/strimzi/kafka/bridge/facades/KafkaFacade.java

```diff
@@ -4,62 +4,65 @@
  */
 package io.strimzi.kafka.bridge.facades;
 
-import io.debezium.kafka.KafkaCluster;
-import io.debezium.util.Testing;
 import io.strimzi.kafka.bridge.utils.KafkaJsonSerializer;
-import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+import kafka.server.KafkaConfig$;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.apache.kafka.streams.integration.utils.EmbeddedKafkaCluster;
 import org.apache.kafka.clients.producer.ProducerRecord;
 import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.io.File;
 import java.io.IOException;
 import java.util.Properties;
+import java.util.UUID;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutionException;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
 
 public class KafkaFacade {
 
-    protected static final int ZOOKEEPER_PORT = 2181;
+    private static final Logger LOGGER = LoggerFactory.getLogger(KafkaFacade.class);
     protected static final int KAFKA_PORT = 9092;
-    protected static final String DATA_DIR = "cluster";
     private static final long OPERATION_TIMEOUT = 60L;
+    protected static EmbeddedKafkaCluster kafkaCluster;
 
-    private static File dataDir;
-    protected static KafkaCluster kafkaCluster;
-
-    private static KafkaCluster kafkaCluster() {
-
-        if (kafkaCluster != null) {
-            throw new IllegalStateException();
-        }
-        dataDir = Testing.Files.createTestingDirectory(DATA_DIR);
+    public static EmbeddedKafkaCluster kafkaCluster() {
 
         Properties props = new Properties();
-        props.put("auto.create.topics.enable", "false");
-
-        kafkaCluster =
-            new KafkaCluster()
-                .usingDirectory(dataDir)
-                .withPorts(ZOOKEEPER_PORT, KAFKA_PORT)
-                .withKafkaConfiguration(props);
+        props.setProperty(KafkaConfig$.MODULE$.AutoCreateTopicsEnableProp(), "false");
```

Comment:
Wouldn't it be better to just use `"auto.create.topics.enable"`? It would be much more readable.